packages/editor-ui/src/App.vue

```diff
@@ -231,6 +233,13 @@ export default defineComponent({
 
 			this.postAuthenticateDone = true;
 		},
+		identifyCurrentUser() {
+			const currentUser = this.usersStore.currentUser;
+			if (currentUser) {
+				const traits: SegmentUserTraits = currentUser.email ? { email: currentUser.email } : {};
+				useSegment().identify(currentUser.id, traits);
```

Comment:
which id is this using? the cloud or app id?

Reply:
also can we get these traits in the store directly instead?

Reply:
Yup.